vss-accessor/src/lib.rs

```diff
@@ -0,0 +1,95 @@
+use ::prost::Message;
+use reqwest;
+use reqwest::Client;
+use std::error::Error;
+
+use crate::vss::{GetObjectRequest, GetObjectResponse, KeyValue, ListKeyVersionsRequest, ListKeyVersionsResponse, PutObjectRequest, PutObjectResponse};
+use crate::vss_error::VssError;
+
+mod vss_error;
+
+pub mod vss {
+    include!(concat!(env!("OUT_DIR"), "/org.vss.rs"));
+}
+
+pub struct VssAccessor {
```

Comment:
Why not `VssClient`? Note that writing the docs often helps with naming.

Reply:
I planned on naming it ServiceAccessor and the lib which contains more auxiliary functions such as secondaryKVStore and versioning as VssClient. 
But i could change it.

Reply:
Hmm... how would this code interact with it? I'd lean more towards calling this `VssClient` if that's what it is as the first words of the docs indicate.